content/stats/measure.md

```diff
@@ -41,8 +41,8 @@ Float64|Records float-like values
 
 ### Source code example
 
-{{<tabs Go Java Python CplusPlus>}}
```

Comment:
Please keep these as they were, no need to radically change a style that has been used throughout, otherwise we have noisy and large PRs